hw/formal/README.md

```diff
@@ -301,6 +302,31 @@ influence for "u_sha2.u_pad.shaf_ren".
 ...
 ```
 
+## How To Run VC Formal
```

Comment:
```suggestion
## Synopsys VC Formal
```

Reply:
I think it should be `### Synopsys VC Formal`

Reply:
I will fix this in the coming PR.